include/devicetree.h

```diff
@@ -463,6 +463,48 @@
  */
 #define DT_NODE_FULL_NAME(node_id) DT_CAT(node_id, _FULL_NAME)
 
+/**
+ * @brief Get the number of node labels a node was given
+ *
+ * Example devicetree fragment:
+ *
+ *     / {
+ *             soc {
+ *                     node1: node2: node3: my-node@12345678 { ... };
+ *             };
+ *     };
+ *
+ * Example usage:
+ *
+ *    DT_NODE_NUM_NODELABEL(DT_NODELABEL(node1)) // 3
+ *
+ * @param node_id node identifier
+ * @retval the number of labels given to the node
+ */
+#define DT_NODE_NUM_NODELABEL(node_id) DT_CAT(node_id, _NODELABEL_NUM)
```

Comment:
rename to `DT_NUM_NODELABELS`

Reply:
What is the reasoning for this? The above and below macros both use the `DT_NODE_` form.

Reply:
Assuming you wanted the `S` postfix instead of the `_NODE_` infix removed, this has been done.